code/datums/statistics/stat_blob.dm

```diff
@@ -18,8 +18,10 @@
 
 /datum/stat_blob/cult/doPostRoundChecks()
 	for(var/datum/mind/M in ticker.minds)
-		if(M.active && istype(M.current, /mob/living/carbon) && M.special_role == "Cultist")
-			surviving_cultists++
+		if(M.active && istype(M.current, /mob/living/carbon))
+			var/mob/living/carbon/C = M.current
+			if(islegacycultist(C))
```

Comment:
Deity's PR that exports stats for dynamic mode info is going to remove this cultist check, not sure if we want to keep your rewritten one or place it elsewhere